creme/creme_config/tests/test_fields.py

```diff
@@ -750,3 +755,116 @@ def test_creation_url_n_allowed(self):
             (reverse('creme_config__add_custom_enum', args=(cfield.id,)), True),
             field.creation_url_n_allowed,
         )
+
+
+class BricksConfigFieldTestCase(CremeTestCase):
+    def test_initial(self):
+        field = BricksConfigField()
+        self.assertEqual(field.initial, {'top': [], 'left': [], 'right': [], 'bottom': []})
+
+        field = BricksConfigField(initial={'top': [], 'left': []})
+        self.assertEqual(field.initial, {'top': [], 'left': []})
+
+    def test_required(self):
+        self.assertRaises(NotImplementedError, BricksConfigField, required=False)
+
+    def test_choices(self):
+        choices = ((1, "a"), (2, "b"), (3, "c"), (4, "d"))
+        field = BricksConfigField(choices=choices)
+
+        choices_list = [(1, "a"), (2, "b"), (3, "c"), (4, "d")]
+        self.assertEqual(field.choices, choices_list)
+        self.assertEqual(field._choices, choices_list)
+        self.assertEqual(field.widget.choices, choices_list)
+
+        choices_set = {1, 2, 3, 4}
+        self.assertEqual(field._valid_choices, choices_set)
+
+    def test_copyfield(self):
+        choices = ((1, "a"), (2, "b"), (3, "c"), (4, "d"))
+        field = BricksConfigField(choices=choices)
+        field_copy = copy.deepcopy(field)
+
+        self.assertFalse(field._choices is field_copy._choices)
+        self.assertFalse(field._valid_choices is field_copy._valid_choices)
+
+    def test_clean_invalid_json(self):
+        choices = ((1, "a"), (2, "b"), (3, "c"), (4, "d"))
+        field = BricksConfigField(choices=choices)
+
+        with self.assertRaises(ValidationError) as context:
+            field.clean('TEST')
+
+        self.assertEqual(context.exception.code, 'invalid')
```

Comment:
If your class inherit 'creme_core.tests.forms.base.FieldTestCase' you get the method 'assertFieldValidationError()'.